lib/plugins/zero-conf/blockcypher/blockcypher.js

```diff
@@ -21,17 +24,32 @@ function authorize (account, toAddress, cryptoAtoms, cryptoCode) {
       })
 
       const confidence = account.confidenceFactor
-
+      const isRBFEnabled = account.rbf
       const url = `https://api.blockcypher.com/v1/btc/main/addrs/${toAddress}?${query}`
 
       return axios.get(url)
         .then(r => {
           const data = r.data
-          const sumTxRefs = txrefs => _.sumBy(txref => highConfidence(confidence, txref), txrefs)
-
-          const authorizedValue = sumTxRefs(data.txrefs) + sumTxRefs(data.unconfirmed_txrefs)
-
-          return cryptoAtoms.lte(authorizedValue)
+          if (isBitcoindAvailable && isRBFEnabled && data.unconfirmed_txrefs) {
+            _.map(unconfirmedTxref => {
+              promise.push(new Promise((resolve, reject) => { resolve(fetchRBF(unconfirmedTxref.tx_hash)) }))
+            }, data.unconfirmed_txrefs)
+            return Promise.all(promise)
+              .then(values => {
+                _.map(rbfInfo => {
+                  _.map(unconfirmedTxref => {
+                    if (rbfInfo.tx_hash === unconfirmedTxref.tx_hash) unconfirmedTxref.rbf = rbfInfo['bip125-replaceable']
+                  }, data.unconfirmed_txrefs)
+                }, values)
```

Comment:
A bit of nitpicking, and mostly for my hate of nested looping unless strictly necessary, but can't this be a `zip` between `values` and `data.unconfirmed_txrefs`, which would result in an array of pairs `[rbfInfo.tx_hash, unconfirmedTxref.tx_hash]` to which you can map a comparing function to each pair.

As I said, a bit of nitpicking, you can easily ignore this, as I don't think the effort would win over the performance boost, but something to consider about readability

Reply:
I did it this way to keep the highConfidence really simple but yeah that nested loop should be avoided. Doing what you suggested I think it requires the highConfidence to change a lot tho, right?

Reply:
Maybe. I'm not the most experienced person with this section of code, so we can pass this and see what Rafael thinks.